recipes/loguru-cpp/recipe.yaml

```diff
@@ -0,0 +1,64 @@
+context:
+  name: loguru-cpp
+  version: 2.1.0.2023.04.06.sha.4adaa18
```

Comment:
@conda-forge/help-c-cpp would a version string similar to this be ok?

The upstream repo does not seem very active.

Reply:
I will assume it's ok to do that since I saw it on other feedstocks such as https://github.com/conda-forge/sam-2-feedstock/blob/8bff6e7d1d520dab1840136ade9b4442cd5b0b2b/recipe/meta.yaml#L3-L4